src/sidebars/star/star.less

```diff
@@ -8,32 +8,30 @@
 
 .confirmation-backdrop {
   cursor: default;
-  position: fixed;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
+  .position(fixed, @top: 0px, @right: 0px, @bottom: 0px, @left: 0px);
   z-index: 1050;
-  display: none;
   .no-overflow;
   outline: 0;
   display: block;
+
   .confirmation-modal {
-    &.flipped{
+    &.flipped {
       .box-shadow(-2px -2px, @dark-background);
```

Comment:
It's currently used 3 times. Once for the search tab (2px 0px - right), once on the top confirmation box (-2px -2px - top left) and on the flipped confirmation box (-2px 2px - bottom left).

These 3 could be abstracted into constants perhaps? Not sure if it's worth it though.